docs/content/get-started/upload-launch.md

```diff
@@ -82,26 +82,27 @@ The next step is to [start a cloud deployment](https://docs.improbable.io/refere
 <br/>
 To launch your cloud deployment:
 
-First enter the parameters in the Deployment Launcher section of the Deployments window. Complete the settings as below:
+1\. First enter the parameters in the **Deployment Launcher** section of the Deployments window. Complete the settings as below:
 <br/>
 
 |Setting name|The setting you add|
 |-----|------|
 |**Assembly Name**| The name you gave the assembly in step 2 - for example; `myassembly`. <br/>(The name needs to conform to the following regex: `[a-zA-Z0-9_.-]{5,64}`.)|
 |**Deployment Name**| A name of your choice - for example; `shootyshooty`. <br/>(The name needs to conform to the following regex: `[a-z0-9_]{2,32}`.)|
-|**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
-|**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
+|**Snapshot File**|`snapshots/cloud.snapshot`<br/>Note that this is a relative path from the root of your SpatialOS project.|
+|**Launch Config File** | `cloud_launch_large.json`<br/>Note that this is a relative path from the root of your SpatialOS project.|
 |**Enable Simulated Players**| Ensure this is checked.|
-|**Override Name**| Leave this checkbox UNchecked.|
+|**Override Name**| Leave this UNchecked.|
 |**Deployment Name**<br/> (The second Deployment Name listed)| Do not edit this field.|
-|**Launch Config File**<br/> (The second config listed)|`cloud_launch_large_sim_players.json`<br/>Note that this is a relative path from the root of your SpatialOS Project.|
+|**Launch Config File**<br/> (The second config listed)|`cloud_launch_large_sim_players.json`<br/>Note that this is a relative path from the root of your SpatialOS project.|
 
 ![]({{assetRoot}}assets/deployment-window.png)
 <br/>_Example completed Deployments window_
 <br/>
 <br/>
-Now start your deployment: Select **Launch deployments**.
+2\. Now start your deployment: Select **Launch deployments**.
 
+**Note:** You will see that there are two Deployment Names and two Launch Config Files listed in the Deployment window; the first listed is your FPS Starter Project, the second is a deloyment for simulated players. You will enable the simulated players in later steps.
```

Comment:
I was in two minds about bolding the setting titles in here - I think probably not as it's information about them rather than "look at them" and we don't want to draw a lot of attention to this but it's a debatable.

I assumed you would "enable" the simulated players but maybe "find out more about" is more accurate?